systems/designsafe/designsafe.storage.frontera.work.json

```diff
@@ -10,6 +10,7 @@
     "canExec": false,
     "notes": {
         "label": "HPC Work",
```

Comment:
Also, should we call this HPC Work (Frontera) ?

Reply:
I don't think so, the work mount is the same on all systems

Reply:
What should happen eventually is use designsafe.storage.work instead of this system, but this is here for backwards compatibility

Reply:
So, I need to hide this in the UI. Can I add notes.isHidden?